### ISSUE TITLE
[.25][HarukaMa] Failed signed message verification caused by space characters

### ISSUE THREAD

**Issue by HarukaMa** [bitshares/bitshares-ui#861]
In `AccountSignedMessages.parseMemo`, message content is being `trim()`ed, possibly making the signature verification fail.

I'm not sure which way is preferred to fix this - stripping the possible line breaks, or `trim` the message _before_ signing? The latter seems bad though, but things would get complicated if the message itself has empty lines.

Claiming this issue (if possible).

A example:
```
-----BEGIN BITSHARES SIGNED MESSAGE-----
test 
-----BEGIN META-----
account=haruka
memokey=BTS7mXAGwyvvouxQ5iDUUgT9DfRBhjCjwhxwZAsVKSsTQyfH2WKjV
block=22612669
timestamp=Wed, 13 Dec 2017 12:21:10 GMT
-----BEGIN SIGNATURE-----
203f328d3cf523d8fd90c7d2bc6395060a53792c4f2ccb1cbdbe47289d9aedecb364f947e1c531e2dbbe22508fe1e76f543a78cfadff0f795f50ce539d322f7e24
-----END BITSHARES SIGNED MESSAGE-----
```

**Comment by wmbutler** [comment]
Placed this into the next Sprint.

**Comment by HarukaMa** [comment]
Assuming the signed message is not tampered by ANY way, I have decided to remove the single `\n` at the start and end of the message which is caused by `split`ing.

**Comment by HarukaMa** [comment]
Already finished in 171219.

**Comment by wmbutler** [comment]
Please link the PR here @HarukaMa

**Comment by HarukaMa** [comment]
Fixed in #864.

Address is haruka (1.2.21561).

**Comment by wmbutler** [comment]
Thanks. You will be paid for this as soon as the foundation pays the ui team.
